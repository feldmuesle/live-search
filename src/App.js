import React from 'react';
import './App.css';

function App() {
  return <div className="App">Exiting stuff to come...</div>;
}

export default App;
