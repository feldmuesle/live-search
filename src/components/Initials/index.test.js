import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import { Initials } from '.';
import '../../App.css';

describe('Initials renders correctly', () => {
  it('renders correctly with given value', () => {
    const component = renderer.create(<Initials firstname="Lisa" lastname="sunshine" />);

    const snapshot = component.toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});

describe('Functionality', () => {
  it('creates intitials correctly fro names', () => {
    render(<Initials firstname="Lisa" lastname="sunshine" />);
    const initials = screen.getByText('LS');

    expect(initials).toBeTruthy();
  });
});
