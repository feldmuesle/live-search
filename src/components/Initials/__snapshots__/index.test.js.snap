// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Initials renders correctly renders correctly with given value 1`] = `
<div
  className="initials"
>
  LS
</div>
`;
